lib/install/validate/config.go

```diff
@@ -46,13 +47,12 @@ func (v *Validator) firewall(ctx context.Context) {
 
 	errMsg := "Firewall check SKIPPED"
 	if !v.sessionValid(errMsg) {
-		return
+		return issues
```

Comment:
don't need to return issues here. You can call v.ListIssues() to get the issues generated in this method